tests/integration/test_vulnerability_detector/test_providers/data/test_cases/cases_update_interval.yaml

```diff
@@ -8,6 +8,27 @@
     provider_name: 'Amazon Linux 1'
     update_interval: '5s'
 
+- name: 'Amazon Linux 2'
```

Comment:
Is it really necessary to add these extra use cases? I think one per operating system for this test is fine.

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/2955/commits/3e908b9a5f383bac61a7c00b9b37e80fd6e7004a